### ISSUE TITLE
os: Testhostname failing on aix-ppc64 CI builder

### ISSUE THREAD

**Issue by pmur** [golang/go#47945]
This test started failing fairly consistently. Is this a machine configuration issue? I am not able to reproduce this on the AIX machine I have access to.

```
--- FAIL: TestHostname (0.02s)
    os_test.go:1766: Hostname() = "power8-aix-host1", want "p8-aix2-golang"
```

https://build.golang.org/log/224243751476e8e9c2c44805d94f54775264c4c7
https://build.golang.org/log/e82c5937a949b2bacbfcda00885061bcdbd89932
...

**Comment by pmur** [comment]
@Helflym fyi

**Comment by Helflym** [comment]
Yes, I'm aware of that. The problem comes from the machine but I need to see it with the maintainers. It should be resolved anytime soon.
